Kidoz-SDK/metabase
# 15s,30s interval selector

### Description

Add 15s and 30s as options for the refresh options

### How to verify

Go to http://metabase-prod-lb-metabase-725947956.us-east-1.elb.amazonaws.com/dashboard/2-example-dashboard

on top right click the clock button to select a refresh interval

you should see 15s and 30s as options

### Demo

![image (3)](https://github.com/Kidoz-SDK/metabase/assets/77276477/97f441f6-29e4-467b-8514-b171927cf1a8)


### Checklist
